erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -147,6 +148,12 @@ def validate_fixed_asset(self):
 					elif asset.status in ("Scrapped", "Cancelled", "Sold"):
 						frappe.throw(_("Row #{0}: Asset {1} cannot be submitted, it is already {2}").format(d.idx, d.asset, asset.status))
 
+	def validate_item_cost_centers(self):
+		for item in self.get("items",None):
```

Comment:
It will give error `None is not iterable` is item table is blank. You can simply use `self.items`.